res/qml/main.qml

```diff
@@ -70,7 +71,7 @@ ApplicationWindow {
                     text: "Prefs"
                     activeColor: Theme.white
                     onClicked: {
-                        Mixxx.PreferencesDialog.show();
+                        Mixxx2.PreferencesDialog.show(); // qmllint disable unqualified
```

Comment:
qmllint complains about this being an unqualified access, which IMO is not correct.